kubel.el

```diff
@@ -256,14 +261,20 @@ CMD is the kubectl command as a list."
     (kubel--append-to-process-buffer
      (format "[%s]\ncommand: %s" process-name str-cmd))))
 
+(defun kubel--env-values ()
+  "Utility function to propgate kubernetes related environment variables."
+  (cl-map 'list (lambda(env) `(:name ,env :value ,(getenv env))) kubel-env-variables))
+
 (defun kubel--exec-to-string (cmd)
   "Replace \"shell-command-to-string\" to log to process buffer.
 
 CMD is the command string to run."
   (kubel--log-command "kubectl-command" cmd)
-  (with-output-to-string
-    (with-current-buffer standard-output
-      (shell-command cmd t "*kubel stderr*"))))
+  (let ((env-values (kubel--env-values)))
```

Comment:
is this the only place where you need to do that? what about `kubel--exec`?

Reply:
This was the only place where it was required for me to unblock it and have a working kubel version in my `envrc` based setup. 

Note that it still isn't perfect and has some shortcomings which I plan to address in a subsequent PR. But I believe those changes are going to be slightly big (I have a working copy locally, but I'm still testing out various scenarios).

Reply:
I suspect things are going to be very wonky if the changes are only applied to this function and not also `kubel--exec`